privacyidea/lib/crypto.py

```diff
@@ -46,12 +46,12 @@
 from .log import log_with
 from .error import HSMException
 import binascii
+import six
```

Comment:
Please add some information to the doc string of this module, that parameter of functions are expected as a certain type and functions in this module usually return a certain type (is it hexlified byte string?)

Reply:
on it.